tutorials/02a_RunningREMINDLocally.md

```diff
@@ -1,10 +1,146 @@
 # Running REMIND locally
-Mika Pflüger (mika.pflueger@pik-potsdam.de)
+Mika Pflüger (mika.pflueger@pik-potsdam.de), Tonn Rüter (tonn.rueter@pik-potsdam.de)
 
-*Note:* This tutorial is a work in progress, not all steps needed are described, yet.
+*Note:* This approach of running REMIND is a work in progress, however successful REMIND runs on local machines have been conducted using this tutorial.
+
+## System Requirements
+
+- Installed python version must be `3.10.X` or older. To check your python version run `python --version` in a terminal of your choice
```

Comment:
Based on what?
Until now, I was only aware on a [3.7 <= $x$ < 3.11 dependency](https://github.com/remindmodel/remind/pull/1466), inherited from `climate-assessment` → `pyam-iamc`.
So both restrictions (here and in the `Makefile`) might contract to just 3.10

Reply:
Based on the `climate-assesment` / `pyam-iamc` dependency. I think python `3.10` would be a reasonable choice

Reply:
Good point. I'll change it to 

Installed python version must be `< 3.11`. We recommend a python version `3.10.X` as it is closest to being up-to-date while remaining REMIND compatible.

Reply:
I updated the whole section to emphasize that most of the System Requirements only concern those willing to run REMIND locally *under Windows*. Is this fine with you, @0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q ?